starfish/core/morphology/Merge/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Algorithms in this module merge multiple BinaryMaskCollections together."""
+from ._base import MergeAlgorithm
+from .simple import SimpleMerge
+
+# autodoc's automodule directive only captures the modules explicitly listed in __all__.
+all_filters = {
```

Comment:
```suggestion
all_methods = {
``` 
? are they all filters?

Reply:
You are correct.  Will fix.